CMS/translations.py

```diff
@@ -754,7 +754,7 @@
         'cy': 'Dewiswyd {} opsiwn'
     },
     'number_selected_institutions': {
-        'en': '{} selected institutions',
```

Comment:
If you're changing how the English works, surely you need to do the same with the welsh?

Reply:
{} have been put back in and replaced with the array length with JS.